tests/src/Functional/UpdatePathTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\Tests\workbench_access\Functional;
+
+use Drupal\FunctionalTests\Update\UpdatePathTestBase;
+
+/**
+ * Defines a class for testing the update path to scheme based access.
+ *
+ * @group workbench_access
+ */
+class UpdatePathTest extends UpdatePathTestBase {
+
+  /**
+   * Set database dump files to be used.
+   */
+  protected function setDatabaseDumpFiles() {
+    $this->databaseDumpFiles = [
+      __DIR__ . '/../../fixtures/workbench_access.update-hook-test.php.gz',
```

Comment:
It's always helpful to commit the script(enabling the module and adding the access_scheme) you ran after installing Drupal.

Reply:
I didn't run a script, I hand configured it 👴